claude-code.el

```diff
@@ -1359,6 +1368,17 @@ MESSAGE is the notification body."
   (claude-code--pulse-modeline)
   (message "%s: %s" title message))
 
+
+(defun claude-code-handle-hook (type buffer-name json-tmpfile &rest args)
```

Comment:
Instead of using a temp file, it'd be nicer if we could capture the stdin from the hook into a shell variable, and then pass it as a string to emacs-client, through to this function.